test/Microsoft.AspNet.Routing.Tests/RouteCollectionTest.cs

```diff
@@ -697,6 +830,14 @@ private static INamedRouter CreateNamedRoute(string name, bool accept = false, s
 
             return target;
         }
+
+        private static RouteOptions GetRouteOptions(bool lowerCaseUrls = false)
+        {
+            var routeOptions = new RouteOptions();
+            routeOptions.LowercaseUrls = lowerCaseUrls;
+            routeOptions.UseBestEffortLinkGeneration = true;
```

Comment:
Correct, my focus here is lower case urls so I decided to hard code it, but wont hurt to take as parameter so others can use the method if they want to in future.